recipes/dbus/1.x.x/conanfile.py

```diff
@@ -104,6 +104,8 @@ def generate(self):
             tc.project_options["checks"] = False
             tc.project_options["doxygen_docs"] = "disabled"
             tc.project_options["modular_tests"] = "disabled"
+            tc.project_options["system_socket"] = self.options.get_safe("system_socket", "")
+            tc.project_options["system_pid_file"] = self.options.get_safe("system_pid_file", "")
```

Comment:
Can you add the equivalent options for CMake below?

Reply:
Done for "system_socket", I couldn't find the cmake option for "system_pid_file" in https://gitlab.freedesktop.org/dbus/dbus/-/blob/master/README.cmake.

Reply:
It seems Dbus can manage an empty string for these options but not "None":
https://gitlab.freedesktop.org/dbus/dbus/-/blob/master/meson.build#L869

Reply:
@Knapock Yeah, there isn't a CMake option for `system_pid_file` since CMake is mostly meant for Windows.

I probably wasn't clear on using `None`. I recommend using that in `options` and `default_options` in the `conanfile.py`. Where you pass it into Meson, pass in `""` as the default value so that it gets passed in when it is `None`.

Reply:
@jwillikers Yes, I gave that a shot yesterday but the builds still failed, complaining about the "None". I pushed that change again.